lib/garage_client/railtie.rb

```diff
@@ -0,0 +1,17 @@
+module GarageClient
+  class Railtie < ::Rails::Railtie
+    initializer :garage_client do |app|
+      RailsInitializer.set_default_name
+    end
```

Comment:
> Is it sure this initializer will be executed after user defined initializer (like config/initializers/garage_client.rb)?

No. I confirmed that by debugging with pry.

>  If not, GarageClient.configuration.options[:name] in L10 will be always nil, I think.

That's true. Actually the check does not make sense but I just want the implementation to be independent to rails' internal.

> Or, if we tolerate overwriting, this initializer can be executed before user defined initializer. In that case, the "name option" will be overwritten by user defined initializer.
> In my opinion, we can tolerate overwriting. So +1 for this implementation.

Since `GarageClient.configuration` is global, different names won't appear by overwriting. Thus I think overwriting is not a problem too.